build/Version.props

```diff
@@ -1,30 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <!-- 
-      Semantic Version. See http://semver.org for full details. 
-      Update for every public release. 
-    -->
-    <SemanticVersionMajor>0</SemanticVersionMajor>
-    <SemanticVersionMinor>1</SemanticVersionMinor>
-    <SemanticVersionPatch>0</SemanticVersionPatch>
-    <!--Valid values: beta1, beta2, EMPTY for stable -->
-    <PreReleaseMilestone>alpha</PreReleaseMilestone>
```

Comment:
I assume minVer does some versioning magic, but we don't seem to set these versions anywhere. So can you help me understand what our versioning story would be with this change?

Reply:
I saw you created https://github.com/open-telemetry/opentelemetry-dotnet/issues/130. Can you please separate it from code cleanups and submit another PR or have a discussion in the #130. 

My concern is that we do not yet have tags in place and I want to make sure build scripts will work nicely with it even before we have tags.

But it shouldn't stop cleanups from merging

Reply:
> Can you please separate it from code cleanups and submit another PR or have a discussion in the #130.

I would love to do that but there were actually quite a few warnings about the way version were defined now, so I thought I'll just go ahead with this change now! 

Minver should be working only with the references to the package set as they are now, we just need to push some sensible tag for it to detect something apart from 0.0.0-alpha.0.offset...